client/state/login/actions.js

```diff
@@ -244,7 +244,19 @@ export const loginSocialUser = ( socialInfo, redirectTo ) => dispatch => {
 			dispatch( {
 				type: SOCIAL_LOGIN_REQUEST_SUCCESS,
 				redirectTo: get( response, 'body.data.redirect_to' ),
+				data: get( response, 'body.data' )
 			} );
+
+			if ( get( response, 'body.data.two_step_notification_sent' ) === 'sms' ) {
```

Comment:
This block is exactly the same as for `loginUser` so we _could_ extract it to a function...